distro/adaptation/centos

```diff
@@ -1,5 +1,6 @@
 apache2: httpd
 apt-file:
+arping: iputils
 autopoint: gettext-devel
```

Comment:
this mapping is different in fedora, which is autopoint: gettext. Need check whether centos can use same package here or not?

Reply:
yes, i have checked this, both gettext and gettext-devel is avilable in centos789, so i keeped one that is already exist in centos.